mobile_app_connector/models/firebase_registration.py

```diff
@@ -37,32 +63,40 @@ def mobile_update_notification_preference(self, json_data, **params):
         :return:
         """
         firebase_id = json_data.get('firebaseId')
-        partner_id = json_data.get('SupporterId'),
+        partner_id = json_data.get('SupporterId')
         reg = self.env['firebase.registration'].search([
             ('registration_id', '=', firebase_id)])
 
+        if firebase_id is None:
+            logging.error(
```

Comment:
```suggestion
            _logger.error(
```
(but first define _logger as it's done in other classes)